tomography_preprocessing/tilt_series_alignment/aretomo.py

```diff
@@ -0,0 +1,48 @@
+from pathlib import Path
+from typing import Optional
+
+import typer
+from lil_aretomo.aretomo import run_aretomo_alignment
+
+from ._cli import cli
+from .. import utils
+
+ARETOMO_ALIGNMENT_COMMAND_NAME = 'AreTomo'
+
+@cli.command(name=ARETOMO_ALIGNMENT_COMMAND_NAME)
+def aretomo_function(
+        tilt_series_star_file: Path = typer.Option(...),
+        output_directory: Path = typer.Option(...),
+        exe: Path = typer.Option(...),	
```

Comment:
I think we should refactor this to `aretomo_executable`

Reply:
Done